templates/experiments/chat/input_bar.html

```diff
@@ -114,19 +147,23 @@ <h4 class="font-bold">File Search Files</h4>
       message: "",
       code_interpreter_files: [],
       file_search_files: [],
+      ocs_attachments: [],
       total_upload_size_bytes: 0,
       disableCodeInterpreterUpload: false,
       init() {
-        let fieldsToWatch = ['message']
+        let fieldsToWatch = ['message'];
         if (this.$refs.fileSearch) {
-          fieldsToWatch.push('file_search_files')
+          fieldsToWatch.push('file_search_files');
         }
         if (this.$refs.codeInterpreter) {
-          fieldsToWatch.push('code_interpreter_files')
+          fieldsToWatch.push('code_interpreter_files');
+        }
+        if (this.$refs.experimentUpload) {
+          fieldsToWatch.push('ocs_attachments');
         }
-        this.$watchMany(fieldsToWatch, (message, cfiles, ffiles) => {
+        this.$watchMany(fieldsToWatch, (message, experimentFiles, cfiles = [], ffiles = []) => {
           const submitButton = document.getElementById('message-submit');
-          if (message || cfiles.length || ffiles.length) {
+          if (message || experimentFiles.length || cfiles.length || ffiles.length) {
```

Comment:
I think this is a bug since the function could get called with a different number of arguments depending on which fields are being watched. In this specific case is still works since the function doesn't change if you assign `cfile` to `experimentFiles` but it would good to fix it anyway to prevent future bugs:


```suggestion
        this.$watchMany(fieldsToWatch, (...args) => {
          const submitButton = document.getElementById('message-submit');
          if (args.some(Boolean)) {
```